### DISCUSSION TITLE
Create a SDSS-like Cluster Cosmology (number counts and mean masses) analysis example.

### DISCUSSION THREAD

**Discussion by yyzhang** [LSSTDESC/firecrown#436] [Ideas]
Issue: create an example to derive cosmology from SDSS-like cluster observables (e.g., https://arxiv.org/pdf/1810.09456) with simplified models. 

This includes creating new ini files, and SACC files to reflect the SDSS cluster observables (number counts and mean masses). 

If there's time, updating the cluster redshift models to allow cluster redshift uncertainties, and include additional cluster MOR relations. 

@m-aguena

**Comment by marcpaterno** [comment]
@yyzhang and @m-aguena is there something needed in the core of Firecrown to enable the production of such an example?
